velox/functions/sparksql/Register.cpp

```diff
@@ -99,18 +99,12 @@ void registerFunctions(const std::string& prefix) {
       prefix + "rlike", re2SearchSignatures(), makeRLike);
   VELOX_REGISTER_VECTOR_FUNCTION(udf_regexp_split, prefix + "split");
 
-  // Subscript operators. See ExtractValue in complexTypeExtractors.scala.
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_subscript, prefix + "getarrayitem");
```

Comment:
Thanks. Should we delete sparksql/Subscript.cpp file as well?

Reply:
Thanks, just fixed it.